packages/synapse-react-client/src/theme/DefaultTheme.tsx

```diff
@@ -10,14 +23,53 @@ export const defaultMuiThemeOptions: ThemeOptions = {
   palette: palette,
   components: {
     MuiAlert: {
+      defaultProps: {
+        square: true,
+        iconMapping: {
+          error: <HighlightOffTwoTone />,
+          info: <InfoTwoTone />,
+          success: <CheckCircleTwoTone />,
+          warning: <ErrorTwoTone />,
+        },
+      },
       styleOverrides: {
         root: ({ ownerState, theme }) => ({
-          borderRadius: '0px',
-          fontSize: '16px',
+          alignItems: 'center',
+          fontSize: '14px',
           borderLeft: `10px solid ${
             // The default severity is 'success'.
             theme.palette[ownerState.severity || 'success']['main']
           }`,
+          color: theme.palette.grey[900],
+        }),
+        icon: ({ theme }) => ({
+          marginRight: theme.spacing(3),
+        }),
+        action: ({ ownerState, theme }) => ({
+          paddingRight: theme.spacing(2),
+          ...(ownerState.onClose &&
+            !ownerState.action && { color: theme.palette.grey[700] }),
+        }),
+        standardSuccess: ({ theme }) => ({
+          backgroundColor: lighten(theme.palette.success.light, 0.8),
+        }),
+        standardError: ({ theme }) => ({
+          backgroundColor: lighten(theme.palette.error.light, 0.8),
+        }),
+        standardWarning: ({ theme }) => ({
+          backgroundColor: lighten(theme.palette.warning.light, 0.8),
+        }),
+        standardInfo: ({ theme }) => ({
+          backgroundColor: lighten(theme.palette.info.light, 0.8),
+        }),
+      },
+    },
+    MuiAlertTitle: {
+      styleOverrides: {
+        root: ({ theme }) => ({
+          color: theme.palette.grey[900],
```

Comment:
We also have `theme.palette.text` but I'm not sure if that object provides the value you need